cmd/gitbase-playground/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 	}
 	defer db.Close()
 
-	static := handler.NewStatic("build", conf.ServerURL)
+	static := handler.NewStatic("build/public", conf.ServerURL)
```

Comment:
I doesn't matter that much because this will change when the frontend stuff is added. But shouldn't this be pointing to `public` now that you moved `index.html` there?

Reply:
nope; as I understood, `public` will contain frontend sources; these front sources will be compiled by `yarn` (next iteration) and the generated code will be stored in `build/public` (to be read by `bindata`)
The logic in behind is to keep the process done in `code-annotation`, but with one extra level depth